src/components/ProvidersDrawer/ProvidersDrawer.tsx

```diff
@@ -34,6 +36,14 @@ function onFileSelected(store: Store) {
   }
 }
 
+function onClearInputs(store: Store) {
+  return () => {
+    store.set('providers')([])
+    store.set('uploadedProvidersFilename')('')
+    store.set('adequacies')({})
```

Comment:
@bcherny updated to explicitly reset adequacies here too. Not great, but not sure what would be better

Reply:
See my comment above - if you decided decide to remove the 'if' in effects.ts, and added benefit is you won't have to add this line here.